modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,3 +1,5 @@
+@import 'balloontoolbar';
```

Comment:
Do we need a separate file for `balloontoolbar` stlyes?

Reply:
I think not. Let's merge it :p